model/encoding/signing_tags.go

```diff
@@ -8,25 +12,25 @@ package encoding
 // a domain tag that specifies the type of the signed object.
 
 func tag(domain string) string {
-	return versionPrefix + domain
+	return protocolPrefix + domain + crypto.BLSCipherSuite
```

Comment:
@Kay-Zee a heads-up this changes will make node signatures be invalid if a spork does not update the code of all nodes in the network